labextension/src/lib/RPCUtils.tsx

```diff
@@ -19,20 +19,41 @@ import { NotebookPanel } from '@jupyterlab/notebook';
 import { Kernel } from '@jupyterlab/services';
 import NotebookUtils from './NotebookUtils';
 import { isError, IError, IOutput } from '@jupyterlab/nbformat';
+import { ToastUtils } from './ToastUtils';
```

Comment:
Why not using the built in Notification API from Juypyter Lab?

See [examples](https://github.com/jupyterlab/extension-examples/tree/main/notifications)

An error Notification will require the following code:

```
import { Notification } from '@jupyterlab/apputils';

//...


// Create an error notification with an action button
Notification.error('Watch out something went wrong.', {
  actions: [
    { label: 'Help', callback: () => alert('This was a fake error.') }
  ],
  autoClose: 3000
});
```